.lycheeignore

```diff
@@ -3,3 +3,6 @@ https://github.com/pharmaverse/admiral/blob/main/
 https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adxx.R
 irongut/CodeCoverageSummary@v1.2.0
 https://packagemanager.rstudio.com/cran/__linux__/focal/latest
+https://raw.githubusercontent.com/pharmaverse/admiral/main/inst/cheatsheet/cheatsheet_image.png
+inst/admiral_cheatsheet.pdf
+inst/admiral_cheatsheet.pptx
```

Comment:
Is this correct?

Reply:
Is there a better way to write this?

Reply:
Do we have any links to `inst/admiral_cheatsheet.pdf` or `inst/admiral_cheatsheet.pptx`?

In the affirmative, should they be changed to `inst/cheatsheet/admiral_cheatsheet.pdf`?